### PR TITLE
Add torchfile recipe

### PR DESCRIPTION
**Pull request by andfoy**

### CODE REVIEW

File: recipes/torchfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (CJ-Wright) [inline recipes/torchfile/meta.yaml]: Please use pip https://conda-forge.org/docs/meta.html#use-pip

File: recipes/torchfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - numpy
```
Comment (CJ-Wright) [inline recipes/torchfile/meta.yaml]: Is python a runtime requirement?

File: recipes/torchfile/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline recipes/torchfile/meta.yaml]: Could this be noarch?

**Comment by synapticarbors** [comment]
@conda-forge-admin, please rerender

**Comment by synapticarbors** [comment]
@andfoy @CJ-Wright I've fixed up what I think were the outstanding issues with this PR. It's ready for a final look and hopefully merged.

**Pull request merged**